remme/ws/constants.py

```diff
@@ -70,4 +92,20 @@ class Status(IntEnum):
     # validator connection failed
     NO_VALIDATOR = 110
 
+    # missing data
+    MISSING_DATA = 111
```

Comment:
Add new statuses to docs

Reply:
It is being added to docs on a separate branch